CHANGELOG.md

```diff
@@ -3,32 +3,34 @@
 ## Changes
 
 - **[Breaking]**: upgrade `eslint-plugin-react-hooks` to `^4.0.2`
-- **[Breaking]**: upgrade `eslint-plugin-flowtype` to `^5.1.0`
-- upgrade `eslint-plugin-react` to `^7.20.0`
+- **[Breaking]**: upgrade `eslint-plugin-flowtype` to `^5.2.0`
+- **[Breaking]**: remove `eslint-plugin-ramda` to `^5.1.0`
+- **[Breaking]**: minimum supported NodeJs version 12.18
+- upgrade `babel-eslint` to `^10.1.0`
+- upgrade `eslint-plugin-eslint-comments` to `^3.2.0`
+- upgrade `eslint-plugin-import` to `^2.22.0`
+- upgrade `eslint-plugin-jsx-a11y` to `^6.3.1`
+- upgrade `eslint-plugin-react` to `^7.21.2`
+- upgrade `eslint-plugin-react-hooks` to `^4.1.2`
```

Comment:
`eslint-plugin-react-hooks` is written above as breaking changes. Could you remove this one and update the breaking changes to `^4.1.2`?